assets/airports/katl.json

```diff
@@ -943,66 +914,133 @@
         }
     },
     "stars": {
-        "CHOPI1": {
-            "icao": "CHOPI1",
-            "name": "Chopi One",
+        "CHPPR1": {
+            "icao": "CHPPR1",
+            "name": "Chopper One",
             "entryPoints": {
-                "DBOLT": ["DBOLT", ["TABBL", "A240+"], ["SEETE", "A190+|S280"], ["BAAKS", "A150+|S280"]],
-                "DDAVS": ["DDAVS", ["TABBL", "A240+"], ["SEETE", "A190+|S280"], ["BAAKS", "A150+|S280"]],
-                "HOWRR": [["HOWRR", "A240+"], ["YORRR", "A210+"], ["RIDZE", "A190+|S280"], ["NICCE", "A150+|S280"]],
-                "MMMOE": ["MMMOE", ["YORRR", "A210+"], ["RIDZE", "A190+|S280"], ["NICCE", "A150+|S280"]],
-                "RAYCL": ["RAYCL", ["SHVRR", "A240+"], ["MEEEE", "A190+|S280"], ["TMBRZ", "A150+|S280"]],
-                "TABBL": [["TABBL", "A240+"], ["SEETE", "A190+|S280"], ["BAAKS", "A150+|S280"]],
-                "TRAYE": ["TRAYE", ["TABBL", "A240+"], ["SEETE", "A190+|S280"], ["BAAKS", "A150+|S280"]]
+                "BBABE": ["BBABE", "LEMKE"],
+                "LEMKE": ["LEMKE"],
+                "MTHEW": ["MTHEW", "MGRIF", "BBABE", "LEMKE"],
+                "RUTTH": ["RUTTH", "JCKEE", "OWNNS"]
             },
-            "body": [["CHOPI", "A150+|S280"], ["ONYON", "A120|S280"]],
+            "body": [],
+            "rwy": {
+                "KATL8L": [["CHPPR", "A130|S250"], "MRCHH", ["^JAGRR", "A90"], "#180"],
+                "KATL8R": [["CHPPR", "A130|S250"], "MRCHH", ["^JAGRR", "A90"], "#180"],
+                "KATL9L": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["RYENN", "A70+"], ["AAKAY", "A60+"]],
+                "KATL9R": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["ANDIY", "A70+"], ["DFINS", "A60+"]],
+                "KATL10": [["CHPPR", "A130|S250"], "MRCHH", ["BSHOP", "A90"], ["NAVVY", "A80+"], ["^DROYD", "A80"], "#180"],
+                "KATL26L": [["CHPPR", "A130"], "MRCHH", "HIDRO", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL26R": [["CHPPR", "A130"], "MRCHH", "HIDRO", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL27L": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL27R": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
+            },
+            "draw": [
+                []

```

Comment:
Why are we not drawing here? Would be nice for future plans, if STARs are drawn.